parser/prism/Translator.cc

```diff
@@ -476,6 +476,37 @@ std::unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
 
             return make_unique<parser::Rational>(parser.translateLocation(loc), value);
         }
+        case PM_REGULAR_EXPRESSION_NODE: {
+            auto regularExpressionNode = reinterpret_cast<pm_regular_expression_node *>(node);
+            pm_location_t *loc = &regularExpressionNode->base.location;
+            pm_location_t *closingLoc = &regularExpressionNode->closing_loc;
+
+            // Sorbet represents the regex content as a vector of string nodes
+            parser::NodeVec parts;
+
+            auto source = parser.extractString(&regularExpressionNode->unescaped);
+            if (source != "") {
```

Comment:
@amomchilov Does this seem like the most reasonable way to make this check? Sorbet does not expect `parts` to contain anything if the regex is empty.